tests/conftest.py

```diff
@@ -73,6 +73,10 @@ def mock_successful_sync_with_wait(respx_mock):
         headers={"Authorization": "Bearer my_api_key"},
     ).mock(
         side_effect=[
+            Response(
+                200,
+                json={"data": {"id": 5, "sync_run_id": 12345, "status": "queued"}},
+            ),
```

Comment:
throwing this fixture state here, as the polling _**should**_ still wait til a completion state is detected